x-pack/plugins/enterprise_search/public/applications/app_search/constants/credentials.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const ADMIN = 'admin';
+export const PRIVATE = 'private';
+export const SEARCH = 'search';
+
+export const TOKEN_TYPE_DESCRIPTION = {
+  [SEARCH]: 'Public Search Keys are used for search endpoints only.',
+  [PRIVATE]: 'Private API Keys are used for read and/or write access on one or more Engines.',
+  [ADMIN]: 'Private Admin Keys are used to interact with the Credentials API.',
+};
```

Comment:
Quick reminder, all user-facing strings should be run through `i18n.translate`:

```js
import { i18n } from '@kbn/i18n';
```

```suggestion
export const TOKEN_TYPE_DESCRIPTION = {
  [SEARCH]: i18n.translate('xpack.enterpriseSearch.appSearch.tokens.search.description', {
    defaultMessage: 'Public Search Keys are used for search endpoints only.',
  }),
  [PRIVATE]: i18n.translate('xpack.enterpriseSearch.appSearch.tokens.private.description', {
    defaultMessage: 'Private API Keys are used for read and/or write access on one or more Engines.',
  }),
  [ADMIN]: i18n.translate('xpack.enterpriseSearch.appSearch.tokens.admin.description', {
    defaultMessage: 'Private Admin Keys are used to interact with the Credentials API.',
  }),
};
```

Reply:
Good catch, thank you.

Reply:
fb3d159a92eae460843b5a78d0d232aa5a259133